app/oshelp/oshelp.go

```diff
@@ -66,6 +67,30 @@ func GetNetrc(os string) string {
 	}
 }
 
+func GetNetrcFile(os string, env map[string]string) *spec.File {
+	netrcName := GetNetrc(os)
+
+	var path string
+	switch os {
+	case OSLinux:
+		path = fmt.Sprintf("/home/ubuntu/%s", netrcName)
+	case OSWindows:
+		path = fmt.Sprintf(`C:\Windows\system32\config\systemprofile\%s`, netrcName)
+	case OSMac:
+		path = fmt.Sprintf("/Users/anka/%s", netrcName)
```

Comment:
are you sure this will be the path everytime? what if the customer is using vm runner that has some other provider than anka?